### ISSUE TITLE
Add small intro/pointer to docs site in crate documentation

### ISSUE THREAD

**Issue by Skirmisher** [etesync/etebase-rs#21]
Having no introduction in the top-level crate documentation is not a very nice experience :( Since the docs site introduces the API nicely in the "guides" section, it would be prudent to include a small welcome message that points people there.

**Comment by tasn** [comment]
We would happily merge a PR that adds this. Wanna add one?

**Comment by Skirmisher** [comment]
I might have an opportunity to at some point, but I created this issue to suggest a mitigation for something that discourages potential users, which might be easily added by a maintainer (in lieu of a full introduction with examples), since I was not in a position to do so immediately.

**Comment by tasn** [comment]
Yeah, I understand! Just saying that this sounds like something that should be easy enough to add, and we would be happy to accept a PR for. Sure, we can do this ourselves, but it's better to get people (especially the originator of the idea) involved. :)
